### PR TITLE
HW 1 PR

### PR DESCRIPTION
**Pull request by ayshajamjam**

### CODE REVIEW

File: aj2604.md [hunk 1]
```diff
@@ -6,14 +6,10 @@ UNI: aj2604
 
 Github: [link](https://github.com/ayshajamjam)
 
+GitHub README: [link](https://github.com/ayshajamjam/ayshajamjam/blob/main/README.md)
 
 ## Project Proposal
-I want to make a chrome extension that can parse through your screen (scrape the text) and determine which vocabulary words you may be unfamiliar with. This can be particularly useful when reading articles online. If you click the extension's button it would then show you the list of words and the definition underneath. You can also save words for future reference.
+- [vocab-extension](https://github.com/ayshajamjam/project-proposals-s2023/blob/main/vocab-extension.md)
```
Comment (timkpaine) [inline aj2604.md:12]: Use relative link like 10am./projects/javascript/vocab-extension.md`

File: aj2604.md [hunk 2]
```diff
@@ -6,14 +6,10 @@ UNI: aj2604
...
 
 ## Project Contributions
-1. NumPy
-2. [MapTalks](https://github.com/maptalks)
-3. [SpaceTime](https://github.com/spencermountain/spacetime)
-4. [Glide](https://github.com/bumptech/glide)
-5. [Stanford-Oval](https://github.com/stanford-oval)
-6. [FreeCodeCamp](https://github.com/freeCodeCamp/freeCodeCamp/issues)
+- [contribution](https://github.com/ayshajamjam/project-proposals-s2023/blob/main/contribution.md)
```
Comment (timkpaine) [inline aj2604.md:15]: Use relative link

File: contribution.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## Project Contributions
```
Comment (timkpaine) [inline contribution.md:1]: Provide description of project according to the homework instructions, put in correct folder (like `projects/python`)

File: vocab-extension.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+## Project Proposal
```
Comment (timkpaine) [inline vocab-extension.md:1]: Move to `projects/javascript/vocab-extension.md`

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by ayshajamjam** [comment]
I have fixed the issues with this PR.

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**